model/clusters_mgmt/v1/machine_type_category_type.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Machine type category
+enum MachineTypeCategory {
+	// General Purpose machine type
```

Comment:
The parser needs the comments to end with a `.` in order to properly generate the openapi.yaml file

Reply:
Fixed. Why is this not covered in the formatter or the UT?

Reply:
Don't know. Can you add it?